.github/workflows/integration.yml

```diff
@@ -13,17 +13,21 @@ jobs:
       - name: Docker Compose Up
         run: docker compose up --build -d
 
+      - name: Configure Materialize
+        run: |
+          docker exec materialized psql -U mz_system -h localhost -p 6877 -d materialize -c 'ALTER SYSTEM SET max_aws_privatelink_connections = 3;'
```

Comment:
If we can't test private link. Do we need to include this?

Reply:
You can still do an integration test that _creates_ the PrivateLink connection (to make sure you got the connection syntax right)! It'll show up in `mz_connections` etc. You just won't be able to create a source that actually uses that PrivateLink connection.

Reply:
I've tried that already but I end up with a TF panic:

```
| 2023-02-22T21:23:10.596Z [INFO]  provider.terraform-provider-materialize: 2023/02/22 21:23:10 [ERROR] could not create connection: CREATE CONNECTION materialize.public.privatelink_conn TO AWS PRIVATELINK (SERVICE NAME 'com.amazonaws.vpce.us-east-1.vpce-svc-0e123abc123198abc',AVAILABILITY ZONES ('use1-az2', 'use1-az6'));: timestamp=2023-02-22T21:23:10.596Z
```

Even though TF manages to create the privatelink connection, it still panics.

My guess is that we are returning an error and TF detects that and fails?

```
materialize=> CREATE CONNECTION materialize.public.privatelink_conn2 TO AWS PRIVATELINK (SERVICE NAME 'com.amazonaws.vpce.us-east-1.vpce-svc-0e123abc123198abc',AVAILABILITY ZONES ('use1-az2', 'use1-az6'));
ERROR:  AWS PrivateLink connections are not supported

materialize=> show connections;
            name             |           type            
-----------------------------+---------------------------
 privatelink_conn            | aws-privatelink
 privatelink_conn2           | aws-privatelink
 ```

Reply:
Ohh, shoot, right, to test PrivateLink connections you need to run with the Kubernetes orchestrator (`--orchestrator=kubernetes`). Bummer. We could get that working with some elbow grease, but it's probably not worth the trouble now.

Reply:
Just removed the `ALTER SYSTEM SET max_aws_privatelink_connections = 3`

We could open a new issue for coming up with a way to test privatelink connections. Maybe we just use the DevEx env for that one test?

Reply:
Created an issue so that we could track this: https://github.com/MaterializeInc/terraform-provider-materialize/issues/41

If you two are ok with this, we could go ahead and merge this one for the moment

Reply:
Don't block on me!